ocp-config/prov-app/Tailorfile.CD

```diff
@@ -0,0 +1,9 @@
+namespace CD
+selector app=doc-gen
+param-file ../../../ods-configuration/ods-core.env
+param-file ../../../ods-configuration/ods-provisioning-app-test.env
+ignore-path bc:/spec/output/to/name
+ignore-path bc:/spec/output/imageLabels
+ignore-unknown-parameters true
+
+bc,is

```

Comment:
I guess here we only need the image stream?

Very likely we also don't need the provisioning app configs ... (and the ignored paths, since they are for the BC)